store/sqlstore/upgrade.go

```diff
@@ -1349,6 +1349,8 @@ func upgradeDatabaseToVersion600(sqlStore *SqlStore) {
 		sqlStore.AlterColumnTypeIfExists("Posts", "FileIds", "text", "varchar(300)")
 	}
 
+	sqlStore.CreateCompositeIndexIfNotExists("idx_jobs_status_type", "Jobs", []string{"Status", "Type"})
```

Comment:
Is this planned for 6.0? If not, I'd suggest creating the 6.1 migration function.

Reply:
It's not. That's why I added the wait tag. I am waiting for 6.0 comment to be removed. But yeah, I can do a 6.1 as well.